modules/ROOT/pages/automated-policies-landing-page.adoc

```diff
@@ -3,21 +3,33 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Automated policies make it easy for Anypoint Platform administrators to satisfy compliance in common security and logging requirements by automatically applying the same set of policies to all of your APIs. +
-This allows API owners to deploy with confidence knowing that the environment in which they are deploying will have critical policies enforced automatically which ensures consistency and reduce possible errors because an administrator no longer needs to remember to insert a policy to enforce common standards manually +
+Automated policies enable Anypoint Platform administrators to comply with common security and logging requirements by automatically applying the same set of policies to all of your APIs. 
+
+This allows API owners to deploy their APIs with confidence, knowing that the environment in which they are deploying will have critical policies enforced automatically which ensures consistency and reduce possible errors because an administrator no longer needs to remember to insert a policy to enforce common standards manually 
+
 API owners can continue to design, build and deploy APIs quickly without compromising security because your best practices are built into the application network.
 
+
+
 All of the xref:policies-ootb-landing-page.adoc[Mule 4 Provided Policies], and any xref:policy-mule3-custom-policies.adoc[Mule 4 Custom Policy] can be configured as automated policies and can be configured to be applied at all the API’s running under an environment.
 
 == Support for Automated Policies
 
+The following rule and guidelines apply to automated policies:
```

Comment:
Is it necessary to specify both one rule and (several?) guidelines?
```suggestion
The following rules apply to automated policies:
```

Reply:
Done! Thanks!